index.js

```diff
@@ -123,6 +123,7 @@ const packages = {
   'ConfigurableMapViewerCMV': 'https://opencollective.com/cmv-app',
   'cypress': 'https://opencollective.com/cypressio',
   'd3-node-editor': 'https://opencollective.com/d3-node-editor',
+  'darcyclarke': 'https://www.patreon.com/darcyclarke',
```

Comment:
After merging I realized this is in the wrong section. This belongs in the username section, not this packages section.

Reply:
Totally gapped on that. Apologies. Thanks for correcting that! 👍